examples/lotr/src/main/scala/com/raphtory/examples/lotr/TutorialRunner.scala

```diff
@@ -55,7 +55,7 @@ object TutorialRunner extends App {
 
   //Connected Components
   graph
-    .at(32674)
+    .range("1/1/2020", "1/1/2021", "1 month 1 week and 1 day")
```

Comment:
Is this working? the range of those dates in milliseconds is probably outside lotr timeline